src/main.rs

```diff
@@ -691,6 +691,51 @@ impl Hardware {
 		self.io_chip_write(Self::MCP23S17_GPPUB, 0xFF);
 	}
 
+	/// Make the BMC produce a sequence of beeps using the Speaker
+	fn bmc_beep(&mut self) -> Result<(), ()> {
+		let seq: &[(u8, u8, u8)] = &[
```

Comment:
It might be worth a note here to explain the units. Possible also to import the neotron-bmc-command crate so you can refer to registers by name and not just by number?